app/models/abilities/everyone.rb

```diff
@@ -29,6 +29,7 @@ def initialize(user)
       can [:search, :comments, :read, :create, :new_comment], Legislation::Annotation
 
       can :read, ::SDG::Goal
+      can :read, ::SDG::Phase
```

Comment:
I noticed there a missing spec for this change.

Reply:
Added :+1:.